storage/innobase/include/os0enc.h

```diff
@@ -31,6 +31,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301  USA
 #define os0enc_h
 
 #include <mysql/components/my_service.h>
+#include "my_aes.h"
```

Comment:
why do we need this header?

Reply:
> why do we need this header?

because of KEYRING_VALIDATION_TAG_SIZE = MY_AES_BLOCK_SIZE;

Reply:
I see